jwql/utils/utils.py

```diff
@@ -596,11 +600,9 @@ def filename_parser(filename):
 
     # Raise error if unable to parse the filename
     except AttributeError:
-        jdox_url = 'https://jwst-docs.stsci.edu/understanding-jwst-data-files/jwst-data-file-naming-conventions'
-        raise ValueError(
-            'Provided file {} does not follow JWST naming conventions.  '
-            'See {} for further information.'.format(filename, jdox_url)
-        )
+        filename_dict = {'recognized_filename': False}
+        logging.exception((f'\nFile; {filename} was not recognized by filename_parser(). Update parser or '
+                           'constants.py if it should be recognized.\n'))
 
```

Comment:
I think my worry here is where this log entry ends up. Are we set up so that calling logging.exception() will go to the log file for whatever monitor is running? If not, should we pass in the log file, pass in the logger instance itself that's already set up to use the file, or do the logging after the function returns?

@bhilbert4 have you run this to see if the log entries show up in the appropriate file?

Reply:
I haven't tried running it from a monitor. Or actually, from archive_database_update nor generate_preview_images, since those are the only two monitor-like things that call it. But I think the filename_parser does need to be flexible enough that it can be called from anywhere, including situations where there is no logger set up (e.g. on the fly). That might argue for doing any logging after the function returns.